TheSuXXStore/src/Controllers/CommentController.php

```diff
@@ -5,39 +5,24 @@
     use Suxx\Commands\CommentFormCommand;
     use Suxx\Http\Request;
     use Suxx\Http\Response;
-    use Suxx\Http\Session;
 
     class CommentController implements Controller
     {
         /**
-         * @var Session
-         */
-        private $session;
-
-        /**
          * @var CommentFormCommand
          */
         private $commentFormCommand;
 
-        public function __construct(
-            Session $session,
-            CommentFormCommand $commentFormCommand)
+        public function __construct(CommentFormCommand $commentFormCommand)
         {
-            $this->session = $session;
             $this->commentFormCommand = $commentFormCommand;
         }
 
         public function execute(Request $request, Response $response)
         {
-            $result = $this->commentFormCommand->execute($request);
-
-            if ($result === false) {
-                header('Location: /suxx/product?pid=' . $request->getValue('product'), 302);
-            }
-
-            $_SESSION = $this->session->getSessionData();
-            header('Location: /suxx/product?pid=' . $request->getValue('product'), 302);
+            $this->commentFormCommand->execute($request);
```

Comment:
weil es in beiden fällen den exaktt selben redirect setzt und somit überflüssig war. also die exakt selbe ausgabe stattfindet. falls ein fehler auftritt wird eine fehlermeldung im view ausgegeben.